components/more_direct_channels/more_direct_channels.tsx

```diff
@@ -367,21 +371,28 @@ export default class MoreDirectChannels extends React.PureComponent<Props, State
                 <div
                     className='more-modal__details'
                 >
-                    <div className='more-modal__name'>
-                        {modalName}
-                        <BotBadge
-                            show={Boolean((option as UserProfileValue).is_bot)}
-                            className='badge-popoverlist'
-                        />
-                        <GuestBadge
-                            show={isGuest(option)}
-                            className='badge-popoverlist'
-                        />
-                    </div>
-                    <div className='more-modal__description'>
-                        {email}
+                    <div className='more-modal__name_email'>
+                        <div className='more-modal__name'>
+                            {modalName}
+                            <BotBadge
+                                show={Boolean((option as UserProfileValue).is_bot)}
+                                className='badge-popoverlist'
+                            />
+                            <GuestBadge
+                                show={isGuest(option)}
+                                className='badge-popoverlist'
+                            />
+                        </div>
+                        <div className='more-modal__description'>
+                            {email}
+                        </div>
                     </div>
                 </div>
+                {!isMobile() &&
+                    <div className='more-modal__lastPostAt'>
+                        <LastPostAt lastPostAt={lastPostAt}/>
```

Comment:
Can we rename this to something like 'UserLastPost' or 'LastPostFromUser' since for me this lives ambiguity.

Reply:
Both of the above names conveys me as a `Post` rather than time. Also why should we use the term `User` ? If we have a Direct channel with `Bot` it doesn't convey the intent. @calebroseland thoughts ?

Reply:
Maybe we can call it <LastPostTimestamp> since this is a variant of a timestamp. And by naming the property 'value' it would be consistent with the original Timestamp and convey the intent.

Reply:
@pradeepmurugesan I think we could probably just use a timestamp directly, idk if we need a unique component for each place a different relative time format is used. (Just my $.02)